data/types/activity/physical/physical_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	dataNormalizer "github.com/tidepool-org/platform/data/normalizer"
 	"github.com/tidepool-org/platform/data/types"
 	"github.com/tidepool-org/platform/data/types/activity/physical"
+	commontypesTest "github.com/tidepool-org/platform/data/types/common/test"
```

Comment:
Looks like other package renaming match the path of the package i.e :  
/data/types/test --> dataTypesTest
so /data/types/common/test should look like dataTypesCommonTest